package.json

```diff
@@ -56,6 +56,7 @@
     "karma-safari-launcher": "^1.0.0",
     "karma-webpack": "^2.0.3",
     "mocha": "^3.4.2",
+    "nyc": "^11.0.2",
```

Comment:
OTOH the newer version of `tap` relies upon `nyc@11` too, so we might just update the existing dependencies.